Gemfile.lock

```diff
@@ -244,4 +244,4 @@ DEPENDENCIES
   wdm (>= 0.1.0)
 
 BUNDLED WITH
-   1.15.1
+   1.15.4
```

Comment:
You can defined the version bundler, thus it not break the current environment of other developers who do not use docker.

`RUN gem install bundler -v 1.15.1`

https://github.com/ember-brasil/website/blob/master/docker-images/ember-guides#L19

Reply:
updated!